frontend/src/component/integrations/IntegrationForm/IntegrationParameters/IntegrationParameter/IntegrationParameter.tsx

```diff
@@ -28,41 +18,32 @@ export const IntegrationParameter = ({
     parametersErrors,
     setParameterValue,
 }: IIntegrationParameterProps) => {
-    const value = config.parameters[definition?.name] || '';
-    const type = resolveType(
-        definition,
-        typeof value === 'string' ? value : ''
-    );
-    const error = parametersErrors[definition.name];
+    const datadogJson = useUiFlag('datadogJsonTemplate');
+    if (config.provider === 'datadog' && definition.name === 'bodyTemplate') {
+        if (!datadogJson) {
+            return null;
+        }
+
+        return (
+            <IntegrationParameterEnableWithDropdown
+                config={config}
+                definition={definition}
+                parametersErrors={parametersErrors}
+                setParameterValue={setParameterValue}
+                enableTitle="Data format"
+                disableOptionText="Standard"
+                enableOptionText="Custom template (JSON)"
```

Comment:
This feels too static. If we want to go with this route, perhaps these labels should be parameterized as well in the addon definition?

Reply:
As per discussions I've removed this part to simplify the UI